SmartDeviceLink-iOS/SmartDeviceLink/SDLChangeRegistration.m

```diff
@@ -55,4 +55,52 @@ - (SDLLanguage *)hmiDisplayLanguage {
     }
 }
 
+- (void)setAppName:(NSString *)appName {
+    if (appName != nil) {
+        parameters[NAMES_appName] = appName;
```

Comment:
@joeljfischer Since you're intending the 'copy' property type above, but then we specify a custom setter method, shouldn't the setter method call copy on the object passed in before storing it?  Same thing for all of the copy properties.